src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -450,11 +452,17 @@ typedef struct uct_ib_mlx5_md {
         uint8_t              dc;
     } dp_ordering_cap;
 
+    /* Striding message receive queue limitations queried from the device */
     struct {
+        /* A bitmap indicating which transports supports striding message based RQ*/
         uint8_t supported_tls;
+        /* Max message size HW limitation in strides*/
```

Comment:
IMO can omit `HW limitation` + `supported by HW`, as it is already stated above (line 455).